docs/handbook/the-shim.md

```diff
@@ -73,6 +73,77 @@ Internally `twind/shim` uses `twind/observe`, which may be useful for advanced u
 <script defer src="https://unpkg.com/twind/shim/shim.umd.js"></script>
 ```
 
+The `twind/shim` module utilizes the {@link twind/observe} module internally, but it provides its own {@link setup} function for customizing the used {@link tw} instance and setting the target node to be shimmed. It also provides a {@link disconnect} function to stop shimming/observing all nodes.
+
+```js
+import { setup, disconnect } from 'twind/shim'
+
+setup({
+  // node element to shim/observe (default: document.documentElement)
+  target: document.querySelector('#__twind'),
```

Comment:
Isn't the `target` property supposed to reference the DOM fragment to observe? (typically the `document.documentElement` or `document.body` element).
In this example, the `#__twind` CSS selector references Twind's own `<style id="__twind"></style>`.

Reply:
😄 You are totally right! Thank you for the report.